src/graphic-device/index.md

```diff
@@ -13,7 +13,7 @@ Used within our wordmark and as a graphic device across all GOV.UK channels, the
 {% grid %}
 
 <div>
-    <img src="./the-dot.svg" alt="The GOV.UK wordmark, with an arrow that points to the dot." class="app-prose-image" loading="lazy" decoding="async" width="669" height="180" style="padding: 30px; max-width: calc(100% - 60px); background-color: #1d70b8;">
+    <img src="./the-dot.svg" alt="The GOV.UK wordmark, with an arrow that points to the dot." loading="lazy" decoding="async" width="669" height="180" style="padding: 30px; max-width: calc(100% - 60px); background-color: #1d70b8;">
```

Comment:
Also not directly related but would be nice if this were turned into Markdown instead and used some other method of applying these styles. (Probably a class on the wrapping div and using the `{% css %}` [Eleventy bundle](https://www.11ty.dev/docs/plugins/bundle/).)